README.md

```diff
@@ -133,6 +127,26 @@ async function handleEvent(event) {
 }
 ```
 
-### `mapRequestToAsset`
+#### `ASSET_NAMESPACE`
+
+type: KV Namespace Binding
+
+The binding name to the KV Namespace populated with key/value entries of files for the Worker to serve. By default, Workers Sites uses a [binding to a Workers KV Namespace](TODO:link-documentation) named `__STATIC_CONTENT`. 
+
+It is further assumed that this namespace consists of static assets such as html, css, javascript, or image files, keyed off of a relative path that roughly matches the assumed url pathname of the incoming request.
+
+```
+return getAssetFromKV(event, { ASSET_NAMESPACE: MY_NAMESPACE })
+```
+ 
+#### `__STATIC_CONTENT_MANIFEST` (optional)
+
+type: text blob (JSON formatted)
+
+The mapping of requested file path to the key stored on Cloudflare.
 
-`mapRequestToAsset` encapsulates the basic logic for converting a url path to a filename. This is the default for the option of the same name passed to `getAssetFromKV`. Good to use as a baseline for custom mappers. This takes as an argument a `Request` object and returns a string representation of the authoritative resource path for use with the Asset Manifest and KV namespace.
+Workers Sites with Wrangler bundles up a text blob that maps request paths to content-hashed keys that are generated by Wrangler as a [cache-busting measure](TODO:link-sites-blog-post). If this option/binding is not present, the function will fallback to using the raw pathname to look up your asset in KV. If, for whatever reason, you have rolled your own implementation of this, you can include your own by passing a stringified JSON object where the keys are expected paths, and the values are the expected keys in your KV namespace.
+
+```
+return getAssetFromKV(event, { __STATIC_CONTENT_MANIFEST: "{\"index.html\": \"index.special.html\"}" })
```

Comment:
Please correct this if it's wrong 🙏

Reply:
i think actually the name of the option is `ASSET_MANIFEST`, which it looks like I goofed in the original. adding suggestions here. other than that, this invocation looks good, except (nit) i'd probably tell a user to `JSON.stringify` an actual object because it is readable.
```suggestion
let assetManifest = { "index.html": "index.special.html" }
return getAssetFromKV(event, { ASSET_MANIFEST: JSON.stringify(assetManifest) })
```